ouroboros-network-testing/src/Ouroboros/Network/Testing/Utils.hs

```diff
@@ -19,16 +19,8 @@ newtype Delay = Delay { getDelay :: DiffTime }
   deriving Show
   deriving newtype (Eq, Ord, Num)
 
-
-genDelayWithPrecision :: Integer -> Gen DiffTime
-genDelayWithPrecision precision = 
-    sized $ \n -> do
-      b <- chooseInteger (1, precision)
-      a <- chooseInteger (0, toInteger n * b)
-      return (fromRational (a % b))
-
 genDelayWithPrecision :: Integer -> Gen DiffTime
-genDelayWithPrecision precision = 
+genDelayWithPrecision precision =
```

Comment:
You can remove this commit together with 79982f5

Reply:
Okay

Reply:
I'm not sure I was able to remove this commit, I tried with `git rebase -i`, do you know any other way?

Reply:
Nvm, got it!